exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -179,6 +179,26 @@ drill.exec: {
     // Use plain Java compilation where available
     prefer_plain_java: false
   },
+  spill: {
```

Comment:
Added "spill" and "hashagg" sections in the override example file, with some comments:

  spill: {
     # These options are common to all spilling operators.
     # They can be overriden, per operator (but this is just for
     # backward compatibility, and may be deprecated in the future)
     directories : [ "/tmp/drill/spill" ],
     fs : "file:///"
  }
  hashagg: {
    # The partitions divide the work inside the hashagg, to ease
    # handling spilling. This initial figure is tuned down when
    # memory is limited.
    #  Setting this option to 1 disables spilling !
    num_partitions: 32,
    spill: {
        # The 2 options below override the common ones
        # they should be deprecated in the future
        directories : [ "/tmp/drill/spill" ],
        fs : "file:///"
    }
  },